### PR TITLE
Automatically infer latest release

### PR DESCRIPTION
**Pull request by neelimamukiri**
Fixes https://github.com/contiv/install/issues/46

### CODE REVIEW

**Comment by neelimamukiri** [comment]
@dojacobs, you can use this script for Jenkins too.

File: scripts/get_latest_release.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# Try to get the latest release
+url=$(curl -s https://api.github.com/repos/contiv/install/releases/latest  | grep -m 1 "browser_download_url")
+
+if [[ "$?" != "0" ]]; then
+  url=$(curl -s https://api.github.com/repos/contiv/install/releases  | grep -m 1 "browser_download_url")
+  if [[ "$?" != "0" ]]; then
+  	exit 1
+  fi
+fi
+release=$(echo $url | awk -F "/" '{print $8}')
```
Comment (dseevr) [inline scripts/get_latest_release.sh]: Can we do something like:

```bash
# (copied from StackOverflow and modified 😄)
python -c 'import json,sys;print json.load(sys.stdin)[0]["assets"][0]["browser_download_url"]')
```

in a function and pipe `curl` into that function to extract the URL?  Using grep + awk to split paths seems somewhat brittle to me.

Comment (neelimamukiri) [inline scripts/get_latest_release.sh]: I've replaced the shell script with a Python script, easier to parse that way. Hopefully Python being installed is a reasonable pre-req.

Comment (dseevr) [inline scripts/get_latest_release.sh]: Awesome!  Debian/Ubuntu and RHEL/CentOS definitely come with it, so I think we're good on that front 😄

**Pull request merged**